index.js

```diff
@@ -1,13 +1,7 @@
 const Clownface = require('./lib/Clownface')
 
-function factory (dataset, nodes) {
-  return new Clownface(dataset, nodes)
-}
-
-factory.dataset = (dataset, nodes) => {
-  console.error(new Error('clownface.dataset(...) is deprecated, just call clownface(...)'))
-
-  return factory(dataset, nodes)
+function factory ({ dataset, graph, term, value, _context }) {
```

Comment:
Maybe a nit, but I would be call a publicly facing parameter with an underscore.

Reply:
The `_context` argument should not show up in the JSDoc, but it's required to support Clownface objects.